openml-h2o/src/test/java/com/feedzai/openml/h2o/H2OModelProviderLoadTest.java

```diff
@@ -142,4 +150,54 @@ public DatasetSchema createDatasetSchema(final Set<String> targetValues) {
                         .build()
         );
     }
+
+    /**
+     * Tests that when loading a model with a given field set A that is linked to a given {@link DatasetSchema} B, where both schemas are not compatible,
+     * the loading fails with a {@link ModelLoadingException}.
+     */
+    @Test
+    public final void testLoadModelWithPartialSchema() {
```

Comment:
Ideally it would be cool to have this test generic in the openml-utils module. However, as this is sort of blocking for us, I'm leaving that to a later time.